src/util/firebase/index.ts

```diff
@@ -34,9 +35,12 @@ const auth = getAuth(app);
 
 const logIn = () => {
   signIn(auth, googleProvider, db);
+
   const upi = auth.currentUser?.email?.split('@')?.[0];
+  const userGroup = new Prando(upi).nextBoolean() ? 'A' : 'B';
   setUserProperties(analytics, {
     upi,
+    userGroup,
```

Comment:
Could you please follow the convention I used for all user properties e.g. `user_group`. This will need to be changed both in the code and the Custom Definitions section in the Firebase console.